contracts/core/CorkConfig.sol

```diff
@@ -388,7 +397,12 @@ contract CorkConfig is AccessControl, Pausable {
     }
 
     function updatePsmRate(Id id, uint256 newRate) external onlyUpdaterOrManager {
-        moduleCore.updateRate(id, newRate);
+        // we update the rate in our provider regardless it's up or down
+        defaultExchangeRateProvider.setRate(id, newRate);
+
+        // we don't bubble up the error since if this is a yield bearing PA, then the value of PA goes up
+        // but we don't want to insure the yield, so we just ignore the error
```

Comment:
seems like an important change to allow this non-upgradable function to support yield bearing PA?